develop/reference/articles/06-themes/03-understanding-the-page-layout.markdown

```diff
@@ -0,0 +1,102 @@
+# Understanding the Page Layout [](id=understanding-the-page-layout)
+
+Understanding the page layout is crucial to knowing what markup to target for 
+styling, how to organize your content, and how to create your site. Your page 
+layout is unique to the requirements and design for your site. The default page 
+layout that the Unstyled theme provides is organized into three key sections:
+
+- **Header**: contains the navigation, site logo and title (if shown), and 
+  sign-in link when the user isn’t logged in
+- **Main Content**: contains the portlets or fragments for the page
+- **Footer**: contains any additional information, such as the copyright or 
+  author
+
+![Figure 1: The page layout is broken into three key sections.](../../images/portal-layout-sections.png)
+
+## Portlets or Fragments [](id=portlets-or-fragments)
+
+The `#content` `Section` makes up the majority of the page. The portlets or 
+fragments are contained inside the `#main-content` `div`. While you can write 
+all of your content manually in your theme templates if you wish, @product@ 
+provides a CMS with several portlets OOTB that you can use to help build your 
```

Comment:
I'm not sure this is an officially accepted description of DXP... `a CMS with several portlets OOTB` doesn't sound very appealing. Is this standard? Do we have a unified way to describe what DXP offers?

Reply:
I guess you're right. This bit is more of a marketing pitch then useful info. I'll see what I can find. Also, I wasn't trying to describe DXP, but the product in general. DXP offers additional features, but a CMS is available in both. The main thing I was trying to emphasize was the portlets that are available (web content display, asset publisher, blogs, etc.) Perhaps I should just say that.

Reply:
LGTM!